components/Head/Head.tsx

```diff
@@ -27,6 +50,7 @@ const Head = ({
   const url = buildCanonicalUrl(router.basePath, propsUrl || router.asPath);
   const title = formatTitle(titleSuffix, propsTitle);
   const description = propsDescription || "";
+  const firstLvlNav = router.asPath.split("/")[1];
```

Comment:
I don't think will give you the right element; perhaps either of these? 
```
  const path = router.asPath.split("/");
  const firstLvlNav = path[path.length-1]
```
or 
```
  const firstLvlNav = router.asPath.split("/").reverse()[0]
```

Reply:
At the absolute path - `https://goteleport.com/docs/enterprise/introduction/` 
`router.asPath` provides a path of the form `/enterprise/introduction/`. 
So next I use a split on "/" and end up with the first level of navigation.

Reply:
Sorry Katya, you're right - I misread this.  I assumed you were checking the query param when I glanced at it.